metadata-ingestion/tests/unit/urns/valid_urns.txt

```diff
@@ -1,13 +1,13 @@
-# Unknown entity types become generic urns
+# Unknown entity types become generic urns (does not apply to Java)
 urn:li:abc:foo
 urn:li:abc:(foo,bar)
 urn:li:abc:(urn:li:dataPlatform:abc,def,prod)
 
 # A bunch of pretty normal urns
 urn:li:corpuser:foo
 urn:li:corpGroup:bar
-urn:li:dataset:(urn:li:dataPlatform:abc,def/ghi,prod)
```

Comment:
@hsheth2 - These are enum values in the pdl schema, they are case sensitive and the urns that do not match casing will fail.

Reply:
yup makes sense - eventually we can add this to python as well